### ISSUE TITLE
Organizations configuration 

### ISSUE THREAD

**Issue by xgp** [p2-inc/keycloak-orgs#248]
Allow global configuration of organizations features. Currently needed are:
- do/don't create org admin user
- allow shared idps
- ?

Path `/realms/{realm}/orgs/config`

Body (also the default values):
```json
{
  "createAdminUserEnabled": true,
  "sharedIdpsEnabled": false
}
```

Methods:
- `GET` shows the config. Requires `view-organizations` `realm-management` role.
- `PUT` updates the config. Requires `manage-organizations` `realm-management` role.

-----

We need to update the `organizationCreation` method in `OrganizationResourceProviderFactory` to check `createAdminUserEnabled` before creating the default organization admin user.

**Comment by xgp** [comment]
Will need a UI? Perhaps a modal off the action menu in orgs list page.

**Comment by xgp** [comment]
@rtufisi This is part of the "shared IdP" organizations work for this week.

**Comment by pnzrr** [comment]
@xgp what's the handling of the issue when the following actions occur: 

1. By default shared IDPs are off. 
2. Shared IDPs are then enabled. 
3. A handful of Orgs are then linked to IDPs and are now _sharing_ IDPs. 
4. Someone wants to disable sharing of IDPs in the global setting. 

How will this use case be handled? 
- Don't allow disabling shared IDPs until all orgs are unlinked from shared IDPs?

**Comment by xgp** [comment]
@pnzrr Unlink all shared IdPs when the config value is set to false.

**Comment by xgp** [comment]
@pnzrr Would also be great to give a warning about this in the UI.

**Comment by pnzrr** [comment]
Added a warning in the other PR

**Comment by xgp** [comment]
fixed in #251
